engine/table/src/main/java/io/deephaven/engine/util/AbstractScriptSession.java

```diff
@@ -110,6 +107,10 @@ protected AbstractScriptSession(
                 .build();
     }
 
+    public void eraseClassCacheDirectory() {
```

Comment:
Add a comment to the constructor, that a class cache directory is created; and should be cleaned up by the caller.

Perhaps we should add a cleanup to the ScriptSession interface.  There are only two of them, the Abstract one and the Delegating one.  That would eliminate instanceof checks in the rest of the code.